snippets/javascript/class-tag-on-scroll.md

```diff
@@ -0,0 +1,81 @@
+# change class tag on scroll
+
+## description
+
+> this snippet will change a preassigned tag to a class on scroll by adding an
+> 'event listener' property
+
+## application
+
+> this snippet is useful when we want to activate an animation or reveal
+> elements on scroll
+
+### code
+
+```javascript
+"use strict";
+
+window.addEventListener("scroll", show);
+
+/* ^ EventListener is the name of the property. - show-  will be the name
+of the second tag */
+
+function show() {
+  let show = document.querySelectorAll(".hide");
+
+  /* ^ hide is the name of the original
+  tag. we add this in the html after the class name. example -
+  class="container hide"*/
+
+  for (let i = 0; i < hide.length; i++) {
+    // ^ here we assign a variable - i - and set a condition for its value
```

Comment:
You forgot something important here: **It will also loop until the condition is true**